src/Polly.Core.Tests/Helpers/FakeTimeProvider.cs

```diff
@@ -1,26 +1,21 @@
 using Moq;
-using Polly.Utils;
 
 namespace Polly.Core.Tests.Helpers;
 
 internal class FakeTimeProvider : Mock<TimeProvider>
 {
     private DateTimeOffset? _time;
 
-    public FakeTimeProvider(long frequency)
```

Comment:
It doesn't help us yet, but do you know if the runtime team are planning on shipping a ready-made fake/mock/stub implementation of `TimeProvider` for tests?

Reply:
Yup, there will be something like that available.